docs/source/tracking.rst

```diff
@@ -133,6 +133,29 @@ an optional ``artifact_path``.
 logged to.
 
 
+Automatic logging
+------------------
+MLflow supports automatic logging from TensorFlow without the need for explicit log
+statements. You can enable this feature by calling :py:func:`mlflow.tensorflow.autolog`
+before your training code.
+
+:py:func:`mlflow.tensorflow.autolog` enables auto-logging from TensorFlow, optionally taking
+a ``metrics_every_n_steps`` argument to specify the frequency with which metrics should be
+logged to MLflow.
+
+The following table details auto-logging capabilities for different TensorFlow workflows:
+
++------------------+--------------------------------------------------------+----------------------------------------------------------+---------------+-----------------------------------+
+| Framework        | Metrics                                                | Parameters                                               | Tags          | Checkpoints                       |
```

Comment:
There's actually a difference between the training checkpoints & the MLflow model that gets logged, I'd add a separate column for the MLflow model. We should also call out that we log Tensorboard logs in the tf.Keras case (maybe another column), it's pretty cool that this feature does that. So our final set of table header columns would be:

Metrics, Parameters, Tags, TensorBoard logs (or maybe still "Checkpoints", idk), and MLflow Model (would be cool to link this table header to https://mlflow.org/docs/latest/models.html)